.cluster.dev/digitalocean-k8s.yaml

```diff
@@ -0,0 +1,21 @@
+cluster:
+  installed: false
+  name: do-test
+  cloud:
+    provider: do
+    region: fra1
+    project: default
+    domain: do.cluster.dev
+  provisioner:
+    type: digitalocean-kubernetes
```

Comment:
provisioner type in digital ocean, could be, in example ```digitalocean-kuberentes, minikube, kops```
We try to keep mapping to original terraform provider resource naming: https://www.terraform.io/docs/providers/do/d/kubernetes_cluster.html
So in terraform its called `digitalocean_kubernetes_cluster`, we'd like to keep this mappings as close.
So next possible provisioners, would be: `minikube`, or `kops`, so it would be less confusing
Regarding:
> Otherwise, provisioner.type: eks with cloud.provider: do - logically valid 
 I also think about this, but to prevent additional complexity, some type of parameters are enabled only if other parameters are set in dependent values. In example if cloud: AWS, we have options for provisioner enabled: eks, minikube, if cloud DO: minikube, digitalocean_kubernetes, etc..